cfgov/cfgov/urls.py

```diff
@@ -150,6 +150,18 @@ def empty_200_response(request, *args, **kwargs):
             flag_name='MMT_MY_MONEY_CALENDAR'),
         name='mmt-my-money-calendar'),
 
+     url(r'^mmt-cost-of-credit-tool/service-worker\.js$',
```

Comment:
Does this app need to work offline? If not, we should get rid of all the service worker stuff. If so, disregard this.

Reply:
I'd like to leave it for now.  I think they are planning on being able to use it offline, but I will check.